app/Resources/views/default/event/_event.html.twig

```diff
@@ -16,22 +16,24 @@
                     <span>Oups, nous n'avons enregistré aucune adhésion pour ton compte. Tu ne pourras pas voter pour cet événement.</span>
                 {% else %}
                     {% set proxy_given = event | givenProxy %}
-                    {% set proxy_received = event | receivedProxy %}
+                    {% set proxy_received = event | receivedProxies %}
                     {% if proxy_given is not null %}
                         {% if proxy_given.owner is null %}
                             <span>Procuration donnée au premier membre volontaire</span>
                         {% else %}
                             <span>Procuration donnée à &nbsp;<b>{{ proxy_given.owner.user.firstname }} {{ proxy_given.owner.user.lastname }}</b></span>
                         {% endif %}
                     {% endif %}
-                    {% if proxy_received is not null %}
-                        {% if proxy_received.giver %}
-                            <span>Procuration portée par <b>{{ proxy_received.owner.user.firstname }}</b> de la part de&nbsp;<b>{{ proxy_received.giver.mainBeneficiary.user.firstname }} {{ proxy_received.giver.mainBeneficiary.user.lastname }}</b></span>
-                        {% else %}
-                            <span><b>{{ proxy_received.owner.user.firstname }}</b> accepte une procuration. <a href="{{ path("event_proxy_lite_remove",{'event':event.id,'proxy':proxy_received.id}) }}" class="red-text">X</a></span>
-                        {% endif %}
+                    {% if proxy_received|length > 0 %}
```

Comment:
pas sur que le `if` soit nécessaire. Si proxy_received est vide, la boucle for d'en dessous n'aura pas lieu ?